src/all.css

```diff
@@ -0,0 +1,18 @@
+* {
+  box-sizing: border-box;
+}
+
+.html {
+  font-family: "Roboto", sans-serif;
+}
+
+.body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+h2,
+p {
+  padding: 0;
+  margin: 0;
+}
```

Comment:
delete this file, you don't use it

Reply:
![image](https://user-images.githubusercontent.com/125824269/222108343-d8a387dc-f3c9-47b4-b72a-d3da0ad2f74a.png)
I delete this file in vs code, i din't know why does it show